example/README.md

```diff
@@ -15,7 +15,7 @@
 3. Start the envoy proxy server
 
    ```
-   docker run --rm -ti --net=host -v $PWD/envoy.yaml:/etc/envoy/envoy.yaml envoyproxy/envoy
+   docker build -t envoy-scalapbexample . && docker run --network host -it envoy-scalapbexample
```

Comment:
For some reason the current approach fails on my computer but it works in this way.